content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,145 @@
+
+**WARNING:** UEFI support in ReactOS is experimental, under heavy development and is not yet ready for the general use.
+We currently cannot give an estimate as to when it will be done, so please stay tuned!
+
+## Shell improvements
+Doug Lyons, Katayama Hirofumi MZ and Carl Bialorucki (cbialorucki) are extensively improving the Shell component of ReactOS to provide a better user experience with the operating system.
+Notably Doug Lyons improved the toolbar icon display in MS Office 2000 and other programs, by fixing some bugs in the internal functions (`SetDIBitsToDevice` and `NtGdiSetDIBitsToDeviceInternal`).
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/before.png" src="/img/blogs/newsletter-103/before.png" caption="BEFORE" >}}
+{{< figure link="/img/blogs/newsletter-103/after.png" src="/img/blogs/newsletter-103/after.png" caption="AFTER" >}}
+{{< /gallery >}}
+
+Katayama improved the ZIP archive file support to allow zipping or unzipping files and folders that have UTF-8 characters in them.
+This will greatly help with the interoperability between systems with different language settings.
+In addition to that, we can choose the code-page of ZIP compression/extraction by using the following registry key (`HKCU\Software\ReactOS:ZipCodePage / UnZipCodePage`).
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/utf8.png" src="/img/blogs/newsletter-103/utf8.png" caption="Extracting from a ZIP file that has Japanese characters on a Chinese locale system">}}
+{{< /gallery >}}
+
+Katayama added an "Internet Browser" icon in the desktop workspace, which will open the default web browser that is installed onto the computer.
+In the future this icon can be hidden through Shell settings.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/internet.png" src="/img/blogs/newsletter-103/internet.png" >}}
+{{< /gallery >}}
+
+Carl added support for large taskbar icons in the Explorer component of the shell.
+This can help with people with visual impairments.
+This feature can be toggled from Taskbar Properties, or through a registry key (`HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced\TaskbarSmallIcons`) by setting a value of 0 (big icons) or 1 (small icons).
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/large.png" src="/img/blogs/newsletter-103/large.png" caption="ReactOS taskbar with large icons">}}
+{{< /gallery >}}
+
+In addition, Carl improved the balloon tooltip to better match the style and consistency with Windows.
+The previous balloon presented artifact issues and could not display the full notification text if it extended over the balloon boundary area.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/before-balloon.png" src="/img/blogs/newsletter-103/before-balloon.png" caption="BEFORE" >}}
+{{< figure link="/img/blogs/newsletter-103/after-balloon.png" src="/img/blogs/newsletter-103/after-balloon.png" caption="AFTER" >}}
+{{< /gallery >}}
```

Comment:
Note: a similar comment may be made there as well:
![image](https://github.com/reactos/web-content/assets/1969829/da3b6a3f-fa49-4406-9ba6-b0fa087f31c9)

and whether it could be tweaked so as it shows similarly to that:
![image](https://github.com/reactos/web-content/assets/1969829/10ad0f84-a503-4686-b274-f6b4f7fb52ec)